package.json

```diff
@@ -673,5 +673,8 @@
 	},
 	"volta": {
 		"node": "20.14.0"
-	}
+	},
+	"enabledApiProposals": [
+		"findFiles2"
```

Comment:
Enables use of proposed new findFiles interface, that automatically excludes files that don't need to be searched.